Debian/bin/lib/Cyrus/Docker/Command/prep.pm

```diff
@@ -0,0 +1,61 @@
+use v5.36.0;
+
+package Cyrus::Docker::Command::prep;
+use Cyrus::Docker -command;
+
+use Process::Status;
+use Term::ANSIColor qw(colored);
+
+my sub run (@args) {
+  say "running: @args";
+  system(@args);
+  Process::Status->assert_ok($args[0]);
+}
+
+sub abstract { 'build a configured cyrus-imapd' }
+
+sub opt_spec {
+  return (
+    [ 'sanitizer' => hidden => { one_of => [
+      [ 'asan'  => 'build with AddressSanitizer' ],
+      [ 'ubsan' => 'build with UBSan' ],
+      [ 'ubsan-trap' => 'build with UBSan and trap on error' ],
+    ] } ],
+    [ 'compiler' => hidden => { one_of => [
+      [ 'gcc' => 'gcc', ],
+      [ 'clang' => 'clang', ],
+    ] } ],
+    [ 'jobs|j=i',    'specify number of parallel jobs (default: 8) to run for make/make check',
+                     { default => 8 },
+    ],
+  );
+}
+
+sub execute ($self, $opt, $args) {
+  my $root = $self->app->repo_root;
+  chdir $root or die "can't chdir to $root: $!";
+
+  my @jobs = ("-j", $self->app->config->{default_jobs} // $opt->jobs);
+
+  run(qw( make lex-fix                  ), @jobs);
+  run(qw( make                          ), @jobs);
+
```

Comment:
Are these lines supposed to be here?  Seems odd to be running make but then a few lines below running configure/build/check/install

Reply:
Those are a mistake, thanks!